### ISSUE TITLE
MendeleyScraper class doesn't implement total_pages which results total = "Unknown" 

### ISSUE THREAD

**Issue by maboelnour** [rayyansys/rayyan-scrapers#25]
`MendeleyScraper` should implement `total_pages` method, as the current implementation uses the `ScraperBase`'s `total_pages` which returns: `Unknown` which breaks the articles importing flow.

**Comment by maboelnour** [comment]
Total pages should be retrieved using `Mendeley-Count` [header](https://dev.mendeley.com/reference/topics/pagination.html)

**Comment by kohrVid** [comment]
Having looked into this, I'm unable to see the `Mendeley-Count` param in endpoints other than `/documents` and the param disappears when the `group_id` param is added.

Apart from Mohammad's link to the pagination docs, there doesn't seem to be much about this in the existing documentation or in any of the official repos (though I did find [this README](https://github.com/Mendeley/api-design-documentation/blob/178bd66682e1ff4ebc960ce1f0a83cff86d28f60/README_old.md#custom-headers) which implies that the header wasn't in use at the time). The python SDK does mention a count for [group members](https://github.com/Mendeley/mendeley-python-sdk/commit/fa2caa879865d8d61f897e6127227cd648b01432#diff-f9c7ee25289b08ea74a522a4d17f7c67aec62d271b096450d30ac715ac6692d7R10-R15) but I think that version of the endpoint might be deprecated as the `Mendeley-Count` didn't appear there either.

I've emailed the support team for help as the missing header blocks this issue
